### PR TITLE
Meson

### PR DESCRIPTION
**Pull request by zerothi**

### CODE REVIEW

File: Makefile.project [hunk 1]
```diff
@@ -57,9 +57,9 @@ install-header: fdict.inc
 
 
 # Force the deletion of both libraries
-clean:
-	-$(RM) $(RM_FLAG_FORCE) $(FDICT_LIB_STATIC) $(FDICT_LIB_SHARED) settings.bash fdict.inc
-
+clean: 
+	-$(RM) $(RM_FLAG_FORCE) $(FDICT_LIB_STATIC) $(FDICT_LIB_SHARED) fdict.inc
+	echo > settings.bash
```
Comment (zerothi) [inline Makefile.project:63]: Why is this change needed?

File: settings.bash [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (zerothi) [inline settings.bash:1]: This should not be there! :)

Comment (jonaslb) [reply settings.bash:1]: It's because i put settings.bash in a `files()` in meson.build - this will error if the file does not exist. So I decided that the "clean state" of the repository would be an empty file instead of a non-existing file. At least if this shouldn't be here the meson.build should be changed.

Comment (zerothi) [reply settings.bash:1]: Ok, I would rather have a work-around for this in meson to not confuse what `settings.bash` should be used for. :)

**Comment by jonaslb** [comment]
should be cleaned up now :)

**Comment by zerothi** [comment]
Thanks @jonaslb
